docs/managing_deis/using-a-ldap-auth.rst

```diff
@@ -0,0 +1,48 @@
+:title: Using a LDAP Auth
+:description: How to configure LDAP to Authenticate users.
+
+.. _using-a-ldap-auth:
+
+Using a LDAP Auth
+=================
+
+Deis Controller supports Single Sign On access control, for now Deis is able to authenticate using LDAP or Active Directory.
+
+Settings used by LDAP
+---------------------
+
+=========================================           =================================================================================
+setting                                             description
```

Comment:
I wrote this doc at controller_settings and moved to a separated doc before. I think that since this is optional, should be equal to the proxy settings. But if its better on controller I can move again.